lib/PTEVal.ml

```diff
@@ -23,12 +23,19 @@ type t = {
   db : int;
   dbm : int;
   el0 : int;
+  attrs: string list;
   }
 
 (* For ordinary tests not to fault, the dirty bit has to be set. *)
-let prot_default =  { oa=""; valid=1; af=1; db=1; dbm=0; el0=1;}
+(* FIXME: Should we set the default attributes to what Linux would use
+   for userspace applications, in that case we would need the
+   A.default_memattr *)
```

Comment:
Ideally `PTEVal.ml` should be arch dependent. That is it should consist in a functor parmetrized by some arch basic structure.
However, the A.default_parameter can be added later, _e.g._ while emitting `pte_<id>` initial write events in `init_writes*` functions in `eventsMonad.ml`.